docs/make.jl

```diff
@@ -1,13 +1,18 @@
+using ApproximateGPs
+
 ### Process examples
 # Always rerun examples
 const EXAMPLES_OUT = joinpath(@__DIR__, "src", "examples")
 ispath(EXAMPLES_OUT) && rm(EXAMPLES_OUT; recursive=true)
 mkpath(EXAMPLES_OUT)
 
+# Obtain path of ApproximateGPs so we can run examples with the same version
+const PATH_ApproximateGPs = pkgdir(ApproximateGPs)
+
 # Install and precompile all packages
 # Workaround for https://github.com/JuliaLang/Pkg.jl/issues/2219
 examples = filter!(isdir, readdir(joinpath(@__DIR__, "..", "examples"); join=true))
-let script = "using Pkg; Pkg.activate(ARGS[1]); Pkg.instantiate()"
+let script = "using Pkg; Pkg.activate(ARGS[1]); Pkg.develop(PackageSpec(; path=relpath(\"$PATH_ApproximateGPs\", pwd()))); Pkg.instantiate()"
```

Comment:
It seems like using a multi-line string works just fine, so how about
```suggestion
let script = """
    using Pkg
    Pkg.activate(ARGS[1])
    Pkg.develop(
        PackageSpec(;
            path=relpath(\"$(pkgdir(ApproximateGPs))\", pwd())
        )
    )
    Pkg.instantiate()
"""
```
? :)
(Thanks for finding `pkgdir` - and I think then using it directly makes more sense than storing it in a constant!)

Reply:
I tried it (as you can see in the commit history) and it didn't work.

Reply:
It looks like it failed because in e880d4c you hadn't yet made the change of 08c076e that fixed the also-existing issue in https://github.com/JuliaGaussianProcesses/ApproximateGPs.jl/pull/106/commits/d0b799715d3849205125ab5679c3438a0f931980 (after you reverted the multi-line `script` variable).

I've tried out my suggestion locally and it works fine now!